app/renderer/components/preferences/payment/enabledContent.js

```diff
@@ -141,7 +141,7 @@ class EnabledContent extends ImmutableComponent {
     let l10nDataId = 'statusNextReconcileDate'
 
     if (!nextReconcileDateRelative) {
-      nextReconcileDateRelative = formattedDateFromTimestamp(moment().add(1, 'months'), 'MMMM Do')
+      nextReconcileDateRelative = formattedDateFromTimestamp(addDays(1, 30), 'MMMM Do')
```

Comment:
This is a change in logic, right? Since February only has 28 days for example. Can you use the add months method instead?
https://date-fns.org/v1.29.0/docs/addMonths

Reply:
ok updated